tests/Microsoft.Identity.Test.Integration/HeadlessTests/UsernamePasswordIntegrationTests.cs

```diff
@@ -84,16 +85,28 @@ public async Task AcquireTokenWithManagedUsernamePasswordAsync()
         [TestCategory("UsernamePasswordIntegrationTests")]
         public async Task AcquireTokenWithFederatedUsernamePasswordAsync()
         {
-            var user = LabUserHelper.GetDefaultUser().User;
+            // Arrange
+            UserQuery query = new UserQuery
+            {
+                FederationProvider = FederationProvider.AdfsV4,
+                IsMamUser = false,
+                IsMfaUser = false,
+                IsFederatedUser = false
```

Comment:
IsFederatedUser = false in the `AcquireTokenWithFederatedUsernamePasswordAsync` method? isn't this the contrary with `AcquireTokenWithManagedUsernamePasswordAsync`?

Reply:
Thx for catching this!